api/graphql/tests/test_reservations/test_reservation_queries.py

```diff
@@ -600,17 +600,17 @@ def test_reservee_date_of_birth_is_not_shown_to_reg_user(self):
         assert_that(PersonalInfoViewLog.objects.count()).is_equal_to(0)
 
     def test_filter_by_reservation_unit_name(self):
-        self.reservation_unit.name_fi = "Koirankoppi"
-        self.reservation_unit.name_en = "Doghouse"
-        self.reservation_unit.name_sv = "Hundkoja"
+        self.reservation_unit.name_fi = "koirankoppi"
```

Comment:
is there a reason for these uncapitalizations?

Reply:
Yeah, to fix the flaky tests. Looks like the database behaves differently in CI because now there has been multiple cases where sorting by string field gets flaky when the field contains data mixed with lowercase and uppercase strings. Works just fine on my machine.

Naturally, the proper way to fix this is to figure out why it behaves differently 😅

Reply:
Related: https://stackoverflow.com/questions/15427529/alphanumeric-case-in-sensitive-sorting-in-postgres

Reply:
I'll merge this now but I created a separate ticket for this: [TILA-2070](https://helsinkisolutionoffice.atlassian.net/browse/TILA-2070). Good task for technical Friyays ☺️

Reply:
Ok, sounds reasonable 👍